src/Shared/Buffers.MemoryPool/MemoryPoolFactory.cs

```diff
@@ -1,12 +1,9 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Buffers;
-
-namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal
+namespace System.Buffers
 {
-    public static class KestrelMemoryPool
+    internal static class MemoryPoolFactory
```

Comment:
Can we give this abetter name and/or namespace?

Reply:
Suggest a name, this is basically unnecessary churn at this point.

Reply:
KestrelMemoryPoolFactory

Reply:
in the old namespace

Reply:
Nah

Reply:
Do you want it approved or not? 😉